src/executeCommandLine/executeCommandLine.ts

```diff
@@ -501,7 +501,7 @@ namespace ts {
         updateSolutionBuilderHost(sys, cb, buildHost);
         const builder = createSolutionBuilder(buildHost, projects, buildOptions);
         const exitStatus = buildOptions.clean ? builder.clean() : builder.build();
-        tracing?.dumpLegend();
+        dumpTracingLegend(); // Will no-op if there hasn't been any tracing

```

Comment:
*sigh*...  I initially thought that maybe it can be more verbose but healthier to do something like

    startTracing();
    tracing?.dumpLegend();
    stopTracing();

but on top of that there's the arguments that are needed for starting, so it's probably not worth it.  But I still dislike that this goes out of the nice only-start-stop-are-exposed thing...

Reply:
I'm a little sad too, but it didn't seem worth reorganizing to address this particular issue.

Also, it specifically doesn't go between start and stop - we start and stop for each project and this goes after the last project (in build mode).